scripts/build/set-project-version.ps1

```diff
@@ -0,0 +1,10 @@
+# Read the project version from the Version.props file
+$versionFilePath = "$env:BUILD_SOURCESDIRECTORY\scripts\version\Version.props"
+Write-Host "Reading the Sonar project version from '${versionFilePath}' ..."
+
+[xml]$versionProps = Get-Content "$versionFilePath"
+$sonarProjectVersion = $versionProps.Project.PropertyGroup.MainVersion
```

Comment:
Let's use a simple variable name, as the `$sonarProjectVersion` confused us this morning
```suggestion
$version = $versionProps.Project.PropertyGroup.MainVersion
```

Reply:
I wanted to rename `$sonarProjectVersion` to `$version`. 
You've renamed `$versionFilePath` instead 😄